README.md

```diff
@@ -1 +1,207 @@
+
+This deployable architecture provides a comprehensive foundation for trust, observability, security, and regulatory compliance by configuring the IBM Cloud account to align with compliance settings, deploying key and secret management services, and deploying the infrastructure to support CI/CD/CC pipelines for secure application lifecycle management. These pipelines facilitate the deployment of the application, vulnerability checks, and auditability, ensuring a secure and trustworthy deployment of Generative AI applications on IBM Cloud.
+
+# Objective and Benefits
+
+This deployable architecture is designed to showcase a fully automated deployment of a WatsonX GenAI through IBM Cloud Project, providing a flexible and customizable foundation for your own Watson-based application deployments on IBM Cloud.
+
+By leveraging this architecture, you can accelerate your deployment and tailor it to meet your unique business needs and enterprise goals.
+
+By using this architecture, you can:
+
+* Establish Trust: The architecture ensures trust by configuring the IBM Cloud account to align with compliance settings as defined in the [Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about) framework.
+* Ensure Observability: The architecture provides observability by deploying services such as IBM Log Analysis, IBM Monitoring, IBM Activity Tracker, and log retention through Cloud Object Storage buckets.
+* Implement Security: The architecture ensures security by deploying IBM Key Protect and IBM Secrets Manager.
+* Achieve Regulatory Compliance: The architecture ensures regulatory compliance by implementing CI/CD/CC pipelines, along with IBM Security Compliance Center (SCC) for secure application lifecycle management.
+
+
+# Deployment Details
+
+To deploy this architecture, follow these steps.
+
+## 1. Prerequisites
+
+Before deploying the deployable architecture, ensure you have:
+
+* Created an API key in the target account with sufficient permissions. The target account is the account that will be hosting the resources deployed by this deployable architecture. See [instructions](https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui) Note the API key, as it will be used later. On evaluation environments, you may simply grant `Administrator` role on `IAM Identity Service`, `All Identity and Access enabled services` and `All Account Management` services. If you need to narrow down further access, for a production environment for instance, the minimum level of permissions is indicated in the [Permission tab](https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/Retrieval_Augmented_Generation_Pattern-5fdd0045-30fc-4013-a8bc-6db9d5447a52-global#permissions) of the deployable architecture.
```

Comment:
I guess we will update these links later?

Reply:
yeah -